cmake/DDCConfig.cmake.in

```diff
@@ -34,4 +34,10 @@ endif()
 
 include(${CMAKE_CURRENT_LIST_DIR}/DDCTargets.cmake)
 
+foreach(target fft;pdi;splines)
```

Comment:
isn't the use of space as separator more readable than semicolon?

Reply:
Fine for me, I will make the change

Reply:
done in 3bda09adbfa6a1c4decec528e060119fbf9aa21b